crates/ruff_linter/src/rules/flake8_pyi/rules/string_or_bytes_too_long.rs

```diff
@@ -67,3 +77,40 @@ pub(crate) fn string_or_bytes_too_long(checker: &mut Checker, string: StringLike
     )));
     checker.diagnostics.push(diagnostic);
 }
+
+fn warnings_dot_deprecated_message<'a>(
+    expr: Option<&'a ast::Expr>,
+    checker: &Checker,
+) -> Option<&'a ast::ExprStringLiteral> {
+    // If `expr` represents a call to `warnings.deprecated()`
+    // (or the `typing_extensions`` backport), return the string-literal
+    // deprecation message passed as the first argument to the call.
+    // Else, return `None`.
+    let Some(ast::Expr::Call(ast::ExprCall {
+        func,
+        arguments: ast::Arguments { args, .. },
+        ..
+    })) = expr
+    else {
+        return None;
+    };
```

Comment:
You can use the `?` operator to simplify some of these returns a little. For example:

```
let call = expr.as_call_expr()?;
semantic.resolve_call_path(call.func.as_ref()).is_some_and(...)
```

Reply:
Oh, of course